pkg/operator/controller/ingress/deployment.go

```diff
@@ -57,6 +58,8 @@ const (
 	RouterCaptureHTTPResponseHeaders = "ROUTER_CAPTURE_HTTP_RESPONSE_HEADERS"
 	RouterCaptureHTTPCookies         = "ROUTER_CAPTURE_HTTP_COOKIE"
 
+	RouterSetOrReplaceOrAppendResponseHeaders = "ROUTER_HTTP_RESPONSE_HEADERS"
```

Comment:
Do you have two different environment variables, one for request headers and one for response headers?  If so, please add both here.  I suggest naming like this:
```suggestion
	RouterEditRequestHeaders = "ROUTER_EDIT_HTTP_REQUEST_HEADERS"
	RouterEditResponseHeaders = "ROUTER_EDIT_HTTP_RESPONSE_HEADERS"
```

Reply:
I don't think adding EDIT is a good option.

Reply:
BTW we don't need to define constant here. I will remove it.

Reply:
defined the constants 	
RouterHTTPResponseHeaders = "ROUTER_HTTP_RESPONSE_HEADERS"
	RouterHTTPRequestHeaders  = "ROUTER_HTTP_REQUEST_HEADERS"